src/components/Settings/UploadButton.tsx

```diff
@@ -27,10 +61,10 @@ export function UploadButton({
       </label>
       <div className="relative flex flex-col bg-black/50 w-[150px] h-[150px] rounded-3xl border-solid border-cerise border-2 mx-auto overflow-hidden">
         <h1 className="relative top-[50%] -translate-y-2/4 text-center text-xl">
-          {!selectedImage ? (
+          {selectedImage == "" ? (
             textInside
           ) : (
-            <img className="mx-auto" src={URL.createObjectURL(selectedImage)} />
+            <img className="mx-auto" src={selectedImage} />
           )}
         </h1>
         <input
```

Comment:
The `::after`-thing isn't visible or clickable in firefox which makes these upload things unusable

Reply:
Should work now.